{
    "id": "CVE-2020-36109",
    "cisaKEV": false,
    "reported_exploited": false,
    "exploit_maturity": "poc",
    "counts": {
        "public_exploit_count": 2
    },
    "timeline": {
        "nvd_published": "2021-02-01"
    },
    "exploits": [
        {
            "url": "https://github.com/tin-z/CVE-2020-36109-POC",
            "name": "PoC DoS CVE-2020-36109",
            "source": "github",
            "date_added": "2021-02-13",
            "exploit_maturity": "poc"
        },
        {
            "url": "https://github.com/sunn1day/CVE-2020-36109-POC",
            "name": "CVE-2020-36109  PoC causing DoS",
            "source": "github",
            "date_added": "2022-04-20",
            "exploit_maturity": "poc"
        }
    ]
}